Prepend instead of append to the module search path, as this script needs to use
the local module build, not the host module build.

Upstream-Status: Submitted
Signed-off-by: Ross Burton <ross.burton@arm.com>

diff --git a/external/vulkancts/scripts/gen_framework.py b/external/vulkancts/scripts/gen_framework.py
index 83c5329ce..10e720910 100755
--- a/external/vulkancts/scripts/gen_framework.py
+++ b/external/vulkancts/scripts/gen_framework.py
@@ -33 +33 @@ from collections import OrderedDict
-sys.path.append(os.path.join(os.path.dirname(__file__), "..", "..", "..", "scripts"))
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "..", "scripts"))
diff --git a/external/vulkancts/scripts/gen_framework_c.py b/external/vulkancts/scripts/gen_framework_c.py
index 95587e642..84b878cb2 100644
--- a/external/vulkancts/scripts/gen_framework_c.py
+++ b/external/vulkancts/scripts/gen_framework_c.py
@@ -28 +28 @@ import re
-sys.path.append(os.path.join(os.path.dirname(__file__), "..", "..", "..", "scripts"))
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "..", "scripts"))
